app/controllers/sites/redesigned_displays_controller.rb

```diff
@@ -3,5 +3,24 @@
 class Sites::RedesignedDisplaysController < Sites::SetupSiteController
   def edit; end
 
-  def update; end
+  def update
+    if @site.update(site_params)
+      redirect_to edit_site_redesigned_display_path(@site),
+                  flash: { success: 'You have updated your font & colors.' }
+    else
+      render :edit
+    end
+  end
+
+  private
+
+  def site_params
+    @site_params = params.require(:site).permit(
```

Comment:
Is the `@site_params` variable needed here?

Reply:
Yep, I was cribbing existing controller code, but 100% not needed.